HIP/hip-1.md

```diff
@@ -90,7 +95,7 @@ The standard HIP workflow is:
 
     Editors are generally quite lenient about this initial review, expecting that problems will be corrected by the reviewing process.
 
-    If the HIP isn’t ready for approval, an editor will send it back to the author for revision, with specific instructions.
+    If the HIP isn’t ready for approval, an editor will send it back to the author for revision, with specific instructions and will change the status of the HIP back to the "Draft".
 
 * Once approved, editors will assign your HIP a number and merge your PR into the main branch.
 
```

Comment:
What status are we updating the HIP to once it is reviewed and merged? I think before it is merged the status is "Review." Is it the same status after the merge?

Reply:
It will be "WIP".
"Draft" is assigned only after a successful review by the editor.
Updated.